packages/ListBoxWithTags/src/ListBoxWithTags.js

```diff
@@ -76,60 +36,65 @@ const renderTrigger = ({
-      aria-haspopup
-      ref={refTrigger}
-      {...propsForTrigger()}
-      onClick={handleClick}
-      onKeyUp={handleKeyUp}
-      onKeyDown={handleKeyDown}
-      size={size}
-      allOptionsAreSelected={allOptionsAreSelected}
-    >
-      <Tags>
-        {selectedOptions.map(item => {
-          if (typeof renderTag === "function") {
-            return renderTag({ option: item, Tag, onRemove: handleRemove(item) });
-          }
-
-          const label = tagLabelKey === null ? item.label : item[tagLabelKey];
-
-          if (typeof label !== "string") {
-            throw Error(
-              `Your item ${JSON.stringify(
-                item
-              )} must include the attribute "label", or you must indicate which attribute should be rendered as the label via the "tagLabelKey" prop.`
+    <>
+      <sc.Trigger
+        {...triggerProps}
+        aria-controls={idListBoxContent}
+        allOptionsAreSelected={allOptionsAreSelected}
+        hasError={hasError}
+        isDisabled={isDisabled || isReadOnly}
+        isReadOnly={isReadOnly}
+        onClick={handleClick}
+        onKeyUp={handleKeyUp}
+        onKeyDown={handleKeyDown}
+        ref={refTrigger}
+        size={size}
+      >
+        <Tags>
+          {selectedOptions.map(item => {
+            if (typeof renderTag === "function") {
+              return renderTag({ option: item, Tag, onRemove: handleRemove(item) });
+            }
+
+            const label = tagLabelKey === null ? item.label : item[tagLabelKey];
+
+            if (typeof label !== "string") {
+              throw Error(
+                `Your item ${JSON.stringify(
+                  item
+                )} must include the attribute "label", or you must indicate which attribute should be rendered as the label via the "tagLabelKey" prop.`
+              );
+            }
+
+            return (
+              <Tag as="li" key={label} onRemove={handleRemove(item)} size={size}>
+                {label}
+              </Tag>
             );
-          }
-
-          return (
-            <Tag as="li" key={label} onRemove={handleRemove(item)} size={size}>
-              {label}
-            </Tag>
-          );
-        })}
-        {selectedOptions.length ? null : (
-          <sc.PlaceHolder>
-            <sc.PlaceHolderText>{t("listBoxWithTags.placeholder")}</sc.PlaceHolderText>
-          </sc.PlaceHolder>
-        )}
-      </Tags>
-      {allOptionsAreSelected ? null : renderCaret()}
-    </sc.Trigger>
+          })}
+          {selectedOptions.length ? null : (
+            <sc.PlaceHolder isDisabled={isDisabled} size={size}>
+              {t("listBoxWithTags.placeholder")}
+            </sc.PlaceHolder>
+          )}
+        </Tags>
+        {allOptionsAreSelected ? null : renderCaret(isDisabled)}
+      </sc.Trigger>
+      <sc.TriggerLabel id={`${triggerProps.id}__label`}>
+        {t("listBoxWithTags.a11y_text_trigger", { options: a11yText })}
+      </sc.TriggerLabel>
```

Comment:
The previous `a11yText` prop (`aria-label` value) is now in this hidden element. The `id` is used along with the `id` for the  element `refLabel` refers to for a better combined accessible name.